environment.sh

```diff
@@ -108,12 +109,14 @@ else
           repo_info=(${line//@/ })
           repo=${repo_info[0]}
           repo_branch=${repo_info[2]}
-          last_update_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq '.[0].commit.committer.date')
+          last_update_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq .[0].commit.committer.date)
```

Comment:
you don't need the single quotes here anymore?

Reply:
No it seems like I never did? This jq parser is causing me a bunch of problems but this is working.